dockers/nvidia/Dockerfile

```diff
@@ -12,26 +12,68 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM nvcr.io/nvidia/pytorch:21.02-py3
+FROM nvcr.io/nvidia/cuda:11.2.2-runtime-ubuntu20.04
```

Comment:
IIRC PyTorch suggested to stay at 11.1 due to some issues with 11.2, so we might want to as well?